### PR TITLE
Add redis tests script and fix some bugs

### PR DESCRIPTION
**Pull request by Lgsdream**

### CODE REVIEW

**Comment by shafreeck** [comment]
Really awesome!!!

File: db/expire.go [hunk 1]
```diff
@@ -159,7 +159,7 @@ func runExpire(db *DB, batchLimit int) {
 		ts := DecodeInt64(rawKey[expireTimestampOffset : expireTimestampOffset+8])
 		if ts > now {
 			if logEnv := zap.L().Check(zap.DebugLevel, "[Expire] not need to expire key"); logEnv != nil {
-				logEnv.Write(zap.Int64("last timestamp", ts))
+				logEnv.Write( zap.String("raw key", string(rawKey)), zap.Int64("last timestamp", ts))
```
Comment (shafreeck) [inline db/expire.go]: The log here is json structured, the key should be a valid json field, use `raw-key`, `last-timestamp` stead

**Comment by shafreeck** [comment]
I tried to run the test but got errors

image

File: db/expire.go (hunk 1 above)
Comment (Lgsdream) [reply db/expire.go]: I have fix it

**Pull request merged**